modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -149,9 +149,6 @@ protected void optionallyMapResourceParameterAsVolume(@NotNull String paramName,
         }
     }
 
-    @Override
-    protected abstract Integer getLivenessCheckPort();
```

Comment:
breaks the binary compatibility

Reply:
This method, with the same signature, is in `GenericContainer`, so I was actually thinking that overriding it here makes no sense. (TBH I didn't release you could override a concrete method with an abstract one, so this gives me headaches 🤣 ). Happy to withdraw if you can see the problem more clearly than I can though.